lib/web_ui/lib/src/engine/embedder.dart

```diff
@@ -62,7 +63,10 @@ class FlutterViewEmbedder {
   /// Abstracts all the DOM manipulations required to embed a Flutter app in an user-supplied `hostElement`.
   final EmbeddingStrategy _embeddingStrategy;
 
-  // The tag name for the root view of the flutter app (glass-pane)
+  // The tag name for the Flutter View, which hosts the app.
+  static const String flutterViewTagName = 'flutter-view';
```

Comment:
@yjbanov can this be `flt-view`? Custom elements work off the tag name, and I wouldn't want to have collisions if we ever implement custom element stuff.